backend/src/main/java/com/google/rolecall/ApplicationLoader.java

```diff
@@ -61,6 +67,86 @@ private void createAdmin() {
     userRepo.save(admin);
     logger.log(Level.WARNING, String.format("Admin User Created: %s %s %s", 
         adminFirstName, adminLastName, adminEmail));
+    createTestData();
+  }
+
+  // Create sample data
+  
+  private enum DataCreateError {
+    OK,
+    OtherError
+  }
+
+  private DataCreateError createOneUser(String fName, String lName, String email, String dateJoined) {
+    Calendar cal = Calendar.getInstance();
+    SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy", Locale.US);
+    User user;
+    try {
+      cal.setTime(sdf.parse(dateJoined));
+
+      user = User.newBuilder()
+          .setFirstName(fName)
+          .setLastName(lName)
+          .setEmail(email)
+          .setDateJoined(cal)
+          .setIsActive(true)
+          .setCanLogin(true)
+          .setIsDancer(true)
+          .build();
+      userRepo.save(user);
+    } catch(InvalidParameterException e) {
+      return DataCreateError.OtherError;
+    } catch(ParseException e) {
+      return DataCreateError.OtherError;
+    }
+    return DataCreateError.OK;
+  }
+
+  private DataCreateError createTestData() {
```

Comment:
I do not think the createTestData method here fits the ApplicationLoader class. There are 2 reasons:

1. All tests related data should in javatests folder. We should not use this in production environment. We have the name but other information including the email and dates are incorrect. The email is an immutable field in the frontend.
2. This will be called every time we have a database refresh and there is no way to opt out and create a clean database.

Some suggestion about the approach:
1. Make the data seeding as part of the javatests and totally remove it from the java folder. (recommended)
2. If we have to seed data, then at least we should mark it with environments and only do it in non-prod.
3. And, we should initiate some controls about when to seed the data, such as creating a flag or some environment variables.

Reply:
Zephyr, you are right about everything you say. Yet, I would prefer to leave the code in for the next few weeks.

We are writing a system for one client and initially maybe two or three users (who will plan the tours of a little over 30 dancers). For now, this is a tiny, one-off system. We want to make the users' startup experience as good as possible. For a non trained typist, entering 31 names with sometimes non standard spelling can be challenging. (I made several mistakes myself when I entered the same data.) Especially given that the names have to be spelled correctly -- and the middle names and suffixes have to be dealt with correctly -- for the picture lookup to work. I believe having the dancers pre loaded will be viewed as a big plus.

Your other point is true too. We need to get the correct email addresses into the system. (Probably not for login purposes, but for notifications.) And given that we don't have them now, and the email field is not editable, we may have to insert the proper email addresses using SQL later.

Anyway, even though this code probably breaks a lot rules for production code, I lobby that we leave it in. The Alvin Ailey project is a very non standard project.

Reply:
I realized the pain point about typing it over and over. How about we do this:

1. Pack this seeding procedure into a dedicated method.
2. Add a TODO saying this is for experimental only and needs to be removed.

Reply:
This is already in a dedicated method. Please ignore 1.

Reply:
Done.

Reply:
Thanks.